modules/apps/frontend-js/frontend-js-recharts/package.json

```diff
@@ -3,7 +3,7 @@
 		"recharts": "1.8.5"
 	},
 	"devDependencies": {
-		"@liferay/npm-bundler": "3.0.1"
+		"@liferay/npm-bundler": "3.1.0-pre.0"
```

Comment:
Why can't we build this using `liferay-npm-scripts` then?

Reply:
That is the project we used to test bundler 3. We did that on purpose and never rolled it back. After this PR I guess it doesn't make sense any more, as we will not be using bundler 3, and it will revert to `npm-scripts` again.